### PR TITLE
Reduce memory usage of View object 

### PR DESCRIPTION
**Pull request by morningman**
ISSUE #1877 
ISSUE #1881

### CODE REVIEW

File: fe/src/main/java/org/apache/doris/catalog/Catalog.java [hunk 1]
```diff
@@ -5126,13 +5126,6 @@ public void createView(CreateViewStmt stmt) throws DdlException {
         View newView = new View(tableId, tableName, columns);
         newView.setComment(stmt.getComment());
         newView.setInlineViewDef(stmt.getInlineViewDef());
-        newView.setOriginalViewDef(stmt.getInlineViewDef());
-        try {
-            newView.init();
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/catalog/Catalog.java]: Do we need to keep this init in order to return parse error ASAP?

Comment (morningman) [reply fe/src/main/java/org/apache/doris/catalog/Catalog.java]: Actually, I don't know why it throws an exception here. the stmt string comes from QueryStmt.toSql()...

Comment (imay) [reply fe/src/main/java/org/apache/doris/catalog/Catalog.java]: Maybe there is some bug in our toSql() implementation

Comment (morningman) [reply fe/src/main/java/org/apache/doris/catalog/Catalog.java]: Add it back

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**